### ISSUE TITLE
How to get additional challenge details get_adhoc_challenges

### ISSUE THREAD

**Issue by aiivers** [cyberjunky/python-garminconnect#124]
I'm trying to use the get_adhoc_challenges endpoint, and the data provided is very limited (see below for response object).
What I'd like to do is find out who is in a challenge and what their step/mileage/count value is. As is, the response doesn't include any information about who is in the challenge. Is there a way to get this information?

{'socialChallengeStatusId': 4, 'socialChallengeActivityTypeId': 4, 'adHocChallengeName': 'Steps Challenge', 'adHocChallengeDesc': 'Steps Challenge', 'ownerUserProfileId': 100633339, 'uuid': '3A039C0C214E499298467FF690EADE2E', 'startDate': '2023-01-23T00:00:00.0', 'endDate': '2023-01-29T23:59:59.0', 'durationTypeId': 0, 'userRanking': 1, 'playerCount': None, 'inviteeCount': None, 'players': [], 'user': 
None, 'alternateChallengeAwards': None, 'socialChallengeType': 2, 'isChallengeCommentDisabled': False}

**Comment by cyberjunky** [comment]
@aiivers can you see this information on the Garmin Connect website somewhere?

**Comment by aiivers** [comment]
@cyberjunky  Yes, if you go to the https://connect.garmin.com/modern/adhoc-challenge/{challenge ID} URL then it gives the information of who is in the challenge and how many steps each person has in a table.

**Comment by cyberjunky** [comment]
@aiivers nice, I will have a look

**Comment by cyberjunky** [comment]
@aiivers but the challenge ID is not something we request already right, i don't see it in your example json
I don't know how to create a adhoc challenge yet lol

**Comment by cyberjunky** [comment]
Closing since stale, please re-open if more info is available.
